### DISCUSSION TITLE
SugarTV Card - How do I add it to a Card

### DISCUSSION THREAD

**Discussion by Atlas-Night-Out** [dwainscheeren/dwains-lovelace-dashboard#789] [General]
I'm seeing about how I could be setting up the SugarTV Card in Hacs

I have installed it but no clue how to add it to Dwains Dashboard or just the Overview Dashboard though I do not have any I want to be using as I find Dwain's a lot easier to get things in place. 

It is supposed to give sugar levels from Dexcom to a display, and on its instructions it only telling me to add these setting!

`type: custom:sugartv-card
value_entity: sensor.dexcom_glucose_value
trend_entity: sensor.dexcom_glucose_trend`

But when I go to Dwains to add a SugarTV Card I get this message showing up, 
No visual editor is available for type 'custom:sugartv-card'.

I do have Nightscout in a browser setup but thought this may be easier to get into Home Assistant.

[https://imgur.com/gallery/7WuyYy1](url)

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
